tests/templates/kuttl/orphaned-resources/03-remove-role-group.yaml.j2

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: hive.stackable.tech/v1alpha1
+kind: HiveCluster
+metadata:
+  name: test-hive
+spec:
+  version: {{ test_scenario['values']['hive-latest'] }}
+  metastore:
+    roleGroups:
+      default:
+        replicas: 1
+        config:
+          database:
+            connString: jdbc:derby:;databaseName=/tmp/hive;create=true
+            user: APP
+            password: mine
+            dbType: derby
```

Comment:
There is no need to repeat parts which should not be changed.

Reply:
done